lang/en.json

```diff
@@ -241,10 +241,10 @@
         }
     },
     "login": {
-        "heading": "Login",
+        "heading": "Log in",
         "field_email": "Email Address:",
         "field_password": "Password:",
```

Comment:
I see there's a lot more instances of this so I'm gonna assume that was intentional and I'm wrong